packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -287,559 +295,688 @@ void main() {
+              ),
+            ),
+          );
 
-      expect(getDecoratorRect(tester).size, const Size(800.0, 56.0));
-      expect(getBorderBottom(tester), 56.0);
-      expect(getBorderWeight(tester), 1.0);
-      final ThemeData theme = Theme.of(tester.element(find.byType(InputDecorator)));
-      expect(getBorderColor(tester), theme.colorScheme.onSurfaceVariant);
-    });
+          // Default outlined text field's container is not filled.
+          expect(findBorderPainter(), paints
+            ..path(
+              style: PaintingStyle.stroke,
+            )
+          );
+        });
 
-    testWidgets('Compliant border when filled and focused', (WidgetTester tester) async {
-      await tester.pumpWidget(
-        buildInputDecorator(
-          isFocused: true,
-          decoration: const InputDecoration(
-            labelText: labelText,
-            filled: true,
-          ),
-        ),
-      );
+        testWidgets('outline has correct height and color', (WidgetTester tester) async {
+          await tester.pumpWidget(
+            buildInputDecorator(
+              isHovering: true,
+              decoration: const InputDecoration(
+                border: OutlineInputBorder(),
+                labelText: labelText,
+                helperText: helperText,
+              ),
+            ),
+          );
 
-      expect(getDecoratorRect(tester).size, const Size(800.0, 56.0));
-      expect(getBorderBottom(tester), 56.0);
-      expect(getBorderWeight(tester), 2.0);
-      final ThemeData theme = Theme.of(tester.element(find.byType(InputDecorator)));
-      expect(getBorderColor(tester), theme.colorScheme.primary);
-    });
+          final ThemeData theme = Theme.of(tester.element(findDecorator()));
+          expect(getBorderColor(tester), theme.colorScheme.onSurface);
+          expect(getBorderWeight(tester), 1.0);
+        });
+      });
 
-    testWidgets('Compliant border when filled and disabled', (WidgetTester tester) async {
-      await tester.pumpWidget(
-        buildInputDecorator(
-          decoration: const InputDecoration(
-            labelText: labelText,
-            enabled: false,
-            filled: true,
+      group('when field is focused', () {
+        testWidgets('container has correct height and shape', (WidgetTester tester) async {
+          await tester.pumpWidget(
+            buildInputDecorator(
+              isFocused: true,
+              decoration: const InputDecoration(
+                border: OutlineInputBorder(),
+                labelText: labelText,
+                helperText: helperText,
+              ),
+            ),
+          );
+
+          expect(getContainerRect(tester).height, containerHeight);
+          expect(getBorder(tester), isA<OutlineInputBorder>());
+          expect(getBorderRadius(tester), const BorderRadius.all(Radius.circular(4.0)));
+        });
+
+        testWidgets('container has correct color', (WidgetTester tester) async {
```

Comment:
Is this one checking the painting style and not the color, as it says in the title?

Reply:
Another good catch, I duplicated this test from similar ones but in this case the container is not filled so the title should be updated. Thanks again!